config/initializers/raven.rb

```diff
@@ -8,6 +8,7 @@
   config.processors << Sentry::Processor::EmailSanitizer
   config.processors << Sentry::Processor::PIISanitizer
   config.processors << Sentry::Processor::LogAsWarning
+  config.processors << Sentry::Processor::FilterRequestBody
```

Comment:
Should this come before the `LogAsWarning` to make sure anything that needs sanitized occurs before the log is written? I haven't done a deep dive into `Sentry::Processor` but the `<<` makes me think its an array-ish object that order can matter.

Reply:
LogAsWarning doesn't write the log, it sets the log level to warning instead of error